packages/jest-snapshot/package.json

```diff
@@ -10,6 +10,10 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "dependencies": {
+    "@babel/core": "^7.2.2",
```

Comment:
`@babel/core` is usually a peer dep (`babel-jest` has it as a peer, for instance). Not sure what's correct in this case, but I'm leaning towards `peer`. Might be difficult to wire through correctly in a pnp setting, though

Reply:
The reason I put it in here is that a vanilla install of jest already depends on `@babel/core`. I’m personally not a fan of peer dependencies generally, and here it doesn’t feel necessary as it it isn’t adding any new dependencies to jest as a whole - and it _is_ a dependency. `toMatchInlineSnapshot` is part of the API, and it depends on babel. 

I tested it by making sure the following works after an `npm init -y` and `npm install jest` in a new directory (this is also how I figured that we didn’t need an extra dependency for the `typescript` or `jsx` plugins):

```javascript
console.log(require.resolve('@babel/core'))
console.log(require.resolve('@babel/parser'))
console.log(require.resolve('@babel/traverse'))
console.log(require.resolve('@babel/generator'))

const {parse} = require('@babel/parser')

console.log(parse('const x: number = 1', {plugins: ['typescript']}))
console.log(parse('const x = () => <div>abc</div>', {plugins: ['jsx']}))
console.log(parse('const x = (): unknown => <div>abc</div>', {plugins: ['typescript', 'jsx']}))
```

Reply:
This should still be a peer so we don't have multiple versions of it.

Maybe add `peerDependenciesMeta` and make it optional?